test/images/image-util.sh

```diff
@@ -136,11 +136,7 @@ build() {
         ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
         # Register qemu-*-static for all supported processors except the current one
         echo "Registering qemu-*-static binaries in the kernel"
-        local sudo=""
-        if [[ $(id -u) != 0 ]]; then
-          sudo=sudo
-        fi
-        ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset -p yes
+        docker run --rm --privileged multiarch/qemu-user-static:5.1.0-2 --reset -p yes
```

Comment:
hmm why are we both curling from their github and pulling from dockerhub?

it makes me very nervous to not pin these.

Reply:
Good point. I've changed this back and simply removed the ``-p yes``. That seems to be working too.

So, it seems that it is like this: that ``-p yes`` is the persistent flag. Only checked this briefly, but if we are using the ``-p yes`` flag then we won't need the curl below or include ``/usr/bin/qemu-$arch-static`` in the images. But that ``-p yes`` is causing the errors:

```Registering qemu-*-static binaries in the kernel
Setting /usr/bin/qemu-alpha-static as binfmt interpreter for alpha
sh: write error: No such file or directory
Setting /usr/bin/qemu-arm-static as binfmt interpreter for arm
sh: write error: No such file or directory
```

errors we're seeing inside the ``gcr.io/k8s-testimages/gcb-docker-gcloud:v20200422-b25d964`` container. In order to use the ``-p yes`` flag, we'd have to use:

```
docker run --rm --privileged multiarch/qemu-user-static:5.1.0-2 --reset -p yes
```

instead of:

```
${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset
```